root_commands.c

```diff
@@ -162,6 +162,12 @@ static void cmd_identify(irc_t *irc, char **cmd)
 		irc->status |= USTATUS_IDENTIFIED;
 		irc_umode_set(irc, "+R", 1);
 
+		if (irc->caps & CAP_SASL) {
+			irc_user_t *iu = irc->user;
+			irc_send_num(irc, 900, "%s!%s@%s %s :You are now logged in as %s",
+				iu->nick, iu->user, iu->host, iu->nick, iu->nick);
+		}
+
```

Comment:
>Should this maybe also be done after registering then?

Hmm, yep, I thought this covered all identification methods but there's another `irc->status |= USTATUS_IDENTIFIED;` in cmd_register(). Worth adding there too.

(BTW, I'm not sure how meaningful this 900 RPL_LOGGEDIN numeric is for clients, especially when it's not the result of AUTHENTICATE. It probably varies between implementations)